### DISCUSSION TITLE
article/spring-boot/how-fast-spring-boot-3-2-virtual-thread

### DISCUSSION THREAD

**Discussion by giscus** [dyc87112/giscus-didispace#10] [Announcements]
# article/spring-boot/how-fast-spring-boot-3-2-virtual-thread

用Spring Boot 3.2虚拟线程搭建静态文件服务器有多快？ Spring Boot 3.2 于 2023 年 11 月大张旗鼓地发布，标志着 Java 开发领域的一个关键时刻。这一突破性的版本引入了一系列革命性的功能，包括： 虚拟线程：利用 Project Loom 的虚拟线程释放可扩展性，从而减少资源消耗并增强并发性。 Native Image支持：通过Native Image编译制作速度极快的应用程序，减少启动时间并优化资源利用率。 JVM 检查点：利用 CRaC 项目的 JVM 检查点机制实现应用程序的快速重启，无需冗长的重新初始化。 RestClient：采用新的 RestClient 接口的功能方法，简化 HTTP 交互并简化代码。 Spring for Apache Pulsar：利用 Apache Pulsar 的强大功能实现强大的消息传递功能，无缝集成到您的 Spring Boot 应用程序中。

https://www.didispace.com/article/spring-boot/how-fast-spring-boot-3-2-virtual-thread.html

**Comment by SodisgaSoft** [comment]
666

**Comment by zanezhangzz** [comment]
期待后续测试虚拟线程，在数据库访问等，存在阻塞的情况下的性能测试
